src/pytom_tm/weights.py

```diff
@@ -609,7 +617,7 @@ def create_ctf(
     tan_term = np.arctan(amplitude_contrast / np.sqrt(1 - amplitude_contrast ** 2))
 
     # determine the ctf
-    ctf = - np.sin(chi + tan_term)
+    ctf = - np.sin(chi + tan_term - np.deg2rad(phase_shift_deg))
```

Comment:
the reference for the convention (https://github.com/dtegunov/tom_deconv) uses `cos` instead of `sin` does that mater for the "0-point" of the phase shift?

Reply:
very good spot, thanks

really need to double check this!

Reply:
You were absolutely right. Below figures illustrating, comparison is a bit weird as the CTF definition in this tom_deconv is exactly inverse. So without the phase shift and same parameters I get:

![ctf_comparison_without_phase_plate](https://github.com/SBC-Utrecht/pytom-match-pick/assets/58044494/ff40b082-6a98-4391-85da-5a7710db2add)

With the phase shift, I indeed need to add the term (instead of subtract) to get the 'correctly' inversed curves:

![ctf_comparison_with_phase_plate](https://github.com/SBC-Utrecht/pytom-match-pick/assets/58044494/e03b6a19-77fe-4393-bc12-199809a97beb)

Reply:
Let me know if it makes sense

Reply:
> Let me know if it makes sense

Certainly does! thanks for checking :+1: